features/node/nto.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Node Tuning Operator related scenarios
+
+Background: Cleaning before if some previous test run failed
+  And I run the :delete admin command with:
+      | object_type       | tuned                                 |
```

Comment:
@skordas 2-spaces indentation for cucumber steps

Reply:
Fixed

Reply:
@skordas we don't use the `Background` directive at all.  I don't know how the system will react to this approach.  If you just want to make sure the object `tuned` is gone, there's an [existing step](https://github.com/openshift/verification-tests/blob/master/features/step_definitions/resources.rb#L128) to do that already.